components/BottomNavigation/tests/snapshot/MDCBottomNavigationBarSnapshotTests.m

```diff
@@ -419,4 +419,34 @@ - (void)testChangeUnselectedIconWhenSelected {
   [self generateAndVerifySnapshot];
 }
 
+- (void)testShadowColorRespondsToDynamicColor {
```

Comment:
Looks like the test didn't capture the snapshot correctly?

Reply:
Done